src/dropdownBadge.js

```diff
@@ -0,0 +1,16 @@
+import setUnreadCount from 'src/util/setUnreadCount.js'
+
+setUnreadCount(0).then(function(res) {
+    // let badge = document.getElementById('notifs')
+    if (res === 0) {
+        badge.removeAttribute("className")
+    } else {
+        // badge.setAttribute("data-badge", res)
+        let badge = document.getElementById('notifs').innerHTML
```

Comment:
In general, you should never access the DOM directly in React-based code (as in the actual HTML via `document` methods). The JSX code in React is just an approximation of the real DOM/HTML, which React will take care of rendering out to real HTML to display in the browser. React calls this the "virtual DOM". Rather you should do whatever calculations and place the result in your JSX code using `state` or `props`, which React will automatically re-render whenever they change.